lib/screens_web/controllers/v2/screen_controller.ex

```diff
@@ -76,6 +76,16 @@ defmodule ScreensWeb.V2.ScreenController do
     |> render("index_multi.html")
   end
 
+  def widget(conn, %{"id" => app_id} = params)
+      when app_id in @app_id_strings do
+    app_id = String.to_existing_atom(app_id)
+
+    conn
+    |> assign(:app_id, app_id)
+    |> assign(:widget_data, if(params["widget"], do: Poison.encode!(params["widget"]), else: nil))
```

Comment:
We already use the `Jason` library for JSON parsing/encoding elsewhere in the application. It's considered to be generally better performance-wise than `Poison`, and we don't need duplicate JSON libraries, so let's stick with that.
```suggestion
    |> assign(:widget_data, if(params["widget"], do: Jason.encode!(params["widget"]), else: nil))
```

Reply:
OHHHhh thank you